### PR TITLE
BD/SP Random Battle: Update Empoleon's movepool

### PR DESCRIPTION
**Pull request by Tarrembeau**

### CODE REVIEW

File: data/mods/gen8bdsp/formats-data.ts [hunk 1]
```diff
@@ -1739,7 +1739,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	empoleon: {
 		tier: "UU",
 		doublesTier: "DUU",
-		randomBattleMoves: ["agility", "defog", "flashcannon", "icebeam", "scald", "stealthrock"],
+		randomBattleMoves: ["agility", "defog", "grassknot", "icebeam", "scald", "stealthrock"],
```
Comment (AnnikaCodes) [inline data/mods/gen8bdsp/formats-data.ts:1742]: Why remove Flash Cannon? It learns it by TM. Is Steel STAB just useless?

Comment (Tarrembeau) [reply data/mods/gen8bdsp/formats-data.ts:1742]: Ice/Steel/Water can't hit water and steel doesn't provide anything over water stab or ice/grass coverage

Comment (Tarrembeau) [reply data/mods/gen8bdsp/formats-data.ts:1742]: Especially since there are close to no fairy mons in the format

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**